### ISSUE TITLE
Is there any pre-trained model for source:de-target:en language pair

### ISSUE THREAD

**Issue by ereday** [pytorch/fairseq#773]
Hi,

I've seen that there are a couple of pre-trained  NMT models exist in the repository for (source:en, target:de) language pair. However, I wonder is there any pre-trained model for the reverse direction, i.e. from source:de to target:en ?

**Comment by myleott** [comment]
We haven't released any De-En models yet. @edunov, do you have any De-En models that would make sense to release?

**Comment by edunov** [comment]
Hi, yes we have just released WMT'19 winning submissions that include De->En models, please check here: https://github.com/pytorch/fairseq/tree/master/examples/wmt19

**Comment by Oxi84** [comment]
Could you also add smaller de en models, such as ones from WMT 2018?

**Comment by atreyasha** [comment]
In case this is of use to anyone, I trained a `de-en` model based following the Scaling NMT [readme](https://github.com/pytorch/fairseq/blob/master/examples/scaling_nmt/README.md) but reversing the translation direction. The model is not particularly great with a `sacrebleu` score of `31.0` on the `newstest2014` test set, but it served my purposes sufficiently.

In case anyone wants to use it, the download link/instructions are available at the repo's [readme](https://github.com/atreyasha/semantic-isometry-nmt#repository-initialization-fire)
